packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -76,6 +80,25 @@ export default /* @ngInject */ ($stateProvider) => {
       ) =>
         isVpsNewRange ? vpsUpgradeTile.getAvailableUpgrades(serviceName) : [],
 
+      isCommitmentAvailable: /* @ngInject */ (ovhFeatureFlipping) =>
+        ovhFeatureFlipping
+          .checkFeatureAvailability(['billing:commitment'])
+          .then((commitmentAvailability) =>
+            commitmentAvailability.isFeatureAvailable('billing:commitment'),
+          )
+          .catch(() => false),
+      goToCommit: /* @ngInject */ ($state) => () =>
+        $state.href('vps.detail.dashboard.commitment'),
+      goToCancelCommit: /* @ngInject */ ($state) => () =>
+        $state.href('vps.detail.dashboard.cancel-commitment'),
+      goToCancelResiliation: /* @ngInject */ ($state) => () =>
+        $state.href('vps.detail.dashboard.cancel-resiliation'),
+      goToResiliation: /* @ngInject */ ($state) => () =>
+        $state.href('vps.detail.dashboard.resiliation'),
+      shouldReengage: /* @ngInject */ (serviceInfo) => () =>
+        serviceInfo.engagedUpTo &&
+        moment(serviceInfo.engagedUpTo).diff(moment(), 'month') < 3,
```

Comment:
As we use a 2API for VPS maybe we could set it in this, that way we can easily change if needed

Reply:
This is now being done in the 2API